src/Components/Selection/Selection.js

```diff
@@ -33,6 +33,7 @@ class Selection extends Component {
   }
 
   render () {
+    const hasMaxChoices = this.state.choices.length >= 3
```

Comment:
glad this works! as far as naming conventions "has" very much looks like a boolean would be the value